examples/taint-checking/taint_checking.metatags

```diff
@@ -0,0 +1,2 @@
+description: This BBE demonstrates how taint checking ensures that programs written in Ballerina are inherently secure.
+keywords: ballerina, ballerina by example, taint checking, security
```

Comment:
Should we consider adding something like "secure programming"? @rdhananjaya

Reply:
+1, in addition, if it's ok we should include taint error messages here as most probably people will initiate a search for taint analysis because they encounter a taint error.

Reply:
What are the error messages?